src/commands/download_folder.ts

```diff
@@ -5,18 +5,31 @@ import { CLIAction } from '../CLICommand/action';
 import {
 	DrivePrivacyParameters,
 	FolderIdParameter,
-	LocalPathParameter,
+	LocalFilePathDownloadParameter,
```

Comment:
Consider refactoring to "DownloadLocalPathParameter"

Reply:
Applied